.github/workflows/rust.yml

```diff
@@ -45,10 +45,10 @@ jobs:
           fetch-depth:             5
           submodules:              recursive
       - name:                      Adding toolchain
-        if:                        matrix.toolchain == 'nightly-2020-07-27'
+        if:                        matrix.toolchain == 'nightly'
```

Comment:
We should get rid of this. I think Parity image which is used as a base has `nightly` already installed. I only added this for a specific version.